### PR TITLE
Touch spells can now be used for high fiving, which casts it on the high-five-ee. 

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

image

https://user-images.githubusercontent.com/51863163/207440026-bbb77b71-1b9a-4330-9192-61b50e079df2.mp4

Offering a touch spell to someone will offer them a high five like with the slapper.

If someone accepts your offer to high five, you will cast the spell on them.

## Why It's Good For The Game

A funny interaction with the offer system. High five people with smite for fantastic results. 

## Changelog

:cl: Melbert
add: You can now high five people with (non-cult) touch spells! Maybe be careful high-fiving a wizard who knows Smite. 
/:cl:

### CODE REVIEW

**Comment by moocowswag** [comment]
Might need to make a policy discussion on this (Is it self antagging to high five a cultist who has stun hand out since they will clearly convert you after)

**Comment by moocowswag** [comment]
I did not read the changelog entry

**Comment by CRITAWAKETS** [comment]
Rock, paper, scissors! Anyone wanna throw? Rock, paper, scissors, on three! One, two, three! _*dies*_

**Comment by Wallemations** [comment]
Slap my hand.

**Comment by Mey-Ha-Zah** [comment]
Does heretic stun or cult stun work?
And would it be possible to make chaplain hand work, and deal light damage to the arm the slap?

**Comment by moocowswag** [comment]
What if the person that accepts the high five has a touch hand active.

**Comment by Overuseddart** [comment]
Would it be possible to make this work for slaps? I want to slap secoffs with EI Nath

**Comment by Mey-Ha-Zah** [comment]
Slap spells would be hilarious

Slap... wait... BOOM

**Comment by Troxonater** [comment]
Common Melbert W.

**Comment by Shadowflame909** [comment]
very epic fun with the wizard

**Comment by OceanFish1** [comment]
Finally I can become Doink the Clown and offer people a highfive with shoch touch

File: code/datums/elements/high_fiver.dm [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	var/open_hands_taker = 0
+	var/slappers_giver = 0
+	// see how many hands the taker has open for high'ing
+	for(var/hand in taker.held_items)
+		if(isnull(hand))
+			open_hands_taker++
+
+	// see how many hands the offerer is using for high'ing
+	for(var/obj/item/slap_check in offerer.held_items)
+		if(slap_check.item_flags & HAND_ITEM)
```
Comment (JohnFulpWillard) [inline code/datums/elements/high_fiver.dm:54]: Are touch spells hand items?

**Pull request merged**